AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -1986,13 +1986,9 @@ - (void)visibilityDidChange:(BOOL)isVisible
 {
 }
 
-/**
- * We currently only set interface state on nodes in table/collection views. For other nodes, if they are
```

Comment:
I'm not sure there is a reason to remove this comment - let's leave it in, at least until it is replaced by something comparable / useful.